src/cache/mod.rs

```diff
@@ -119,6 +119,19 @@ impl Cache {
         Ok(p)
     }
 
+    /// Get problem from name
```

Comment:
Maybe it's more good about adding the fuzzy search, not only the problem name. As strict matching sometimes work hard.

Reply:
And you can try `leetcode list` to get the name of question, as for me, this feature is a little duplicated :)

Reply:
I'll look into adding some fuzzy searching to make the feature a bit better. Thanks for the review!